internal/wasm/wasm.go

```diff
@@ -24,6 +29,19 @@ type InitMetadata struct {
 	// retrieved via HTTP. In these cases, no filesystem will be mounted.
 	URL string `mapstructure:"url"`
 
+	// StrictSandbox when true uses fake sources to avoid vulnerabilities such
+	// as timing attacks.
+	//
+	// # Affected configuration
+	//
+	//   - sys.Walltime increments with a constant value when read, initially a
+	//     second resolution of the current system time.
+	//   - sys.Nanotime increments with a constant value when read, initially
+	//     zero.
+	//   - sys.Nanosleep returns immediately.
+	//   - Random number generators are seeded with a deterministic source.
+	StrictSandbox bool `mapstructure:"strictSandbox"`
```

Comment:
Do you think this should be on by default, or off by default?

Reply:
TL;DR; I recommend starting with strict off until at least global config is available.

It is early enough that we can turn it on by default, but maybe we should get the higher level wiring in? otherwise, we'll get people having to make the fine grained config to turn it off vs teaching them to use the new knob discussed in https://github.com/dapr/dapr/issues/6422

Even though this avoids some problems, there will be questions like we get in slack. Like why does something run too fast in a bad way?  (sleep is turned off) Why is the date wrong? things like this. So if we turn it more strict, we should have our config story sorted out first.